src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,104 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 1 << 4;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] table;
+    private int threshold;
+    private int size;
 
     @Override
     public void put(K key, V value) {
-
+        if (++size >= threshold || table.length == 0) {
```

Comment:
You may remove " || table.length == 0" . 
And instead of it create a constructor and initialize "table" and "threshold" in constructor.